DeckBuilder/Source/Deck.cpp

```diff
@@ -9,4 +9,20 @@ Deck::Deck()
     std::vector <Card> deck;
 }
 
+void Deck::addCard(Card &cardToAdd) {
+    deck.push_back(cardToAdd);
+}
+
+void Deck::removeCard(Card &cardToremove) {
+    
```

Comment:
ta metoda specjalnie jest pusta ?

Reply:
Jeszcze będziemy ją robić. Do końca tygodnia raczej zrobimy + poprawimy to o czym wspomniałeś